src/pages/typescript/using-wrappers-and-decorators.mdx

```diff
@@ -0,0 +1,118 @@
+import { Callout } from "nextra-theme-docs"
+
+# Using wrappers and decorators
+
+## Node.js
+
+Use the Autometrics wrappers and decorators to instrument the functions you want
+to track. Usually it makes sense to wrap functions that are doing some business
+logic, making network calls, or are otherwise critical to the service (e.g.:
+request handlers or database calls).
+
+### Wrapping plain-old functions
+
+Wrappers are simple functions that wrap the original function declaration and
+instrument it with metrics. They allow the language service plugin to add
+additional information in the type docs.
+
+Use function wrappers to wrap request handlers, database calls, or other
+pieces of important business logic that you want to measure.
+
+<Callout type="info">
+    Wrapped functions must be named. Autometrics will, not affect the wrapped
```

Comment:
"Autometrics will, not..." this doesn't make sense

Also might be worth saying something like "named (as opposed to anonymous arrow functions)"